packages/lib-classifier/src/store/UserProjectPreferencesStore.js

```diff
@@ -28,22 +28,40 @@ const UserProjectPreferencesStore = types
       addDisposer(self, projectDisposer)
     }
 
-    function createTempUPPOrFetchUPP () {
+    function createTempUPP () {
+      const tempUPP = UserProjectPreferences.create({ id: 'guestPreferencesDoNotPost', preferences: {} })
```

Comment:
Do we still need this? PFE doesn't use it any more.

Reply:
I don't know - @srallen can probably speak to this one